src/rosdiscover/recover/loader.py

```diff
@@ -211,6 +219,8 @@ def _load_value(self, dict_: t.Mapping[str, t.Any]) -> SymbolicValue:
             return self._load_reads_param(dict_)
         elif kind == "reads-param-with-default":
             return self._load_reads_param_with_default(dict_)
+        elif kind == "varRef":
```

Comment:
code smell: all other kinds use a form of snakecase (with `-` instead of _`), but `varRef` switches to camelCase

Reply:
fixed